src/react-apps/applications/runtime/src/features/form/workflow/reducer/index.ts

```diff
@@ -30,10 +30,13 @@ const WorkflowReducer: Reducer<IWorkflowState> = (
 
   switch (action.type) {
     case ActionTypes.GET_CURRENT_STATE_FULFILLED: {
-      const workFlowState = (action as IGetCurrentStateFulfilled).state;
+      const workflowState = (action as IGetCurrentStateFulfilled).state;
+      if (workflowState === WorkflowSteps.Archived) {
+        document.body.className = 'a-bgGreenLight flex-column d-flex';
+      }
```

Comment:
This should be done in the saga, not in the reducer.
The reducer should ONLY update the redux-state.

Reply:
Fixed